internal/db/schema/migrations/oss/postgres/56/02_add_data_key_foreign_key_references.up.sql

```diff
@@ -30,6 +30,17 @@ begin;
         references kms_data_key_version (private_id)
         on delete restrict
         on update cascade;
+  -- Adding a foreign key reference to the kms key means we have
+  -- to delete the session when the scope is deleted, or else
+  -- we can't cascade scope deletions to the key referenced here.
+  alter table session
+    drop constraint iam_scope_project_fkey;
+  alter table session
+    add constraint iam_scope_project_fkey
+      foreign key (project_id)
+        references iam_scope_project (scope_id)
+        on delete cascade
+        on update cascade;
```

Comment:
I added this to an existing migration file rather creating a completely new one, since the change above this is what necessitates this change. Let me know if you'd rather I move this to a separate file.